.github/workflows/forest.yml

```diff
@@ -68,13 +68,7 @@ jobs:
         uses: mozilla-actions/sccache-action@v0.0.4
         timeout-minutes: '${{ fromJSON(env.CACHE_TIMEOUT_MINUTES) }}'
         continue-on-error: true
-      - name: Install Apt Dependencies
-        run: |
-          sudo make install-deps
       - name: Cargo Install
-        env:
-          # To minimize compile times: https://nnethercote.github.io/perf-book/build-configuration.html#minimizing-compile-times
-          RUSTFLAGS: "-C linker=clang -C link-arg=-fuse-ld=lld"
```

Comment:
I think clang is preinstalled, and this RUSTFLAGS saves ~1min linking time.

https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md#language-and-runtime

Reply:
According to docs it should, though the build failed on missing dependency because of this. Let me reproduce it.

Reply:
Ah I see, I guess it failed because the llvm toolchain is missing (required by `cc`)

Reply:
Hm, actually there is no error. I'll re-run it a few times just in case.